myproject/tests.py

```diff
@@ -1,29 +1,58 @@
-import unittest
+import pytest
 
 from pyramid import testing
 
+ITEMS_TO_CHECK = [
+    'title',
+    'creation_date',
+    'body'
+]
 
-class ViewTests(unittest.TestCase):
-    def setUp(self):
-        self.config = testing.setUp()
 
-    def tearDown(self):
-        testing.tearDown()
+def test_list_view():
+    from .views import list_view
+    request = testing.DummyRequest()
+    info = list_view(request)
+    assert 'title' in info
```

Comment:
I would imagine that you'd want the list view to return a list of entries, in which case I'd expect this case to fail, since it appears to believe that only one entry will come back out from the view when it is called.